qa-ibd.planx-pla.net/portal/gitops.json

```diff
@@ -424,6 +437,46 @@
       "dropdowns": {}
     }
   ],
+  "studyViewerConfig": [
+    {
+      "dataType": "study_viewer_file",
+      "title": "Studies",
+      "titleField": "title",
+      "rowAccessor": "name",
+      "listItemConfig": {
+        "blockFields": ["description"],
+        "tableFields": ["name", "code", "title", "principle_investigator", "publisher","study_design_primary_purpose","study_phase","verification_date"]
+      },
+      "singleItemConfig": {
+        "blockFields": ["description"],
+        "tableFields": ["name", "code", "title", "principle_investigator", "publisher","study_design_allocation","study_design_primary_purpose","study_phase","verification_date","source","rights"],
+        "sideBoxes": []
+      },
+      "fieldMapping": [
+        { "field": "title", "name": "Study Title" },
+        { "field": "principle_investigator", "name": "Principle Investigator"},
+        { "field": "study_design_allocation", "name": "Design Allocation"},
+        { "field": "study_design_primary_purpose", "name": "Primary Purpose"},
+        { "field": "verification_date", "name": "Verification Date"}
+      ],
+    "buttons": [
+        {
+          "type": "download"
```

Comment:
we won't need a download button since we're not linking files to the study viewer